ckanext/canada/plugin/public_plugin.py

```diff
@@ -216,54 +216,47 @@ def get_blueprint(self) -> List[Blueprint]:
         return [canada_views]
 
     def _canada_citation_map(self, cite_data: DataDict,
-                             pkg_dict: DataDict) -> Dict[str, Any]:
+                             pkg_dict: DataDict):
         lang = 'en'
-        if has_request_context():
+        try:
             lang = h.lang()
+        except Exception:
```

Comment:
always better to use the a more specific exception class so that this doesn't hide some other problem

Reply:
Yeah the lang helper tries to get current language from the flask environ which does not exist in the CLI of course. Exception is `RuntimeError`